ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-run-kueue-e2e-tests.robot

```diff
@@ -12,7 +12,7 @@ Resource          ../../../../tests/Resources/Page/OCPLogin/OCPLogin.robot
 *** Variables ***
 ${KUEUE_KUBECONFIG}         %{WORKSPACE=.}/kueue-kubeconfig
 ${WORKER_NODE}              ${EMPTY}
-${KUEUE_RELEASE_ASSETS}     %{KUEUE_RELEASE_ASSETS=https://github.com/opendatahub-io/kueue/releases/latest/download}
+${KUEUE_RELEASE_ASSETS}     %{KUEUE_RELEASE_ASSETS=https://github.com/opendatahub-io/kueue/releases/download/v0.8.1-1}
```

Comment:
I think currently we have v0.8.1-2. Is there any reason to not use it?

Reply:
Was it used for verification of 2.14?
I see it was built 5 days ago, so I wasn't sure if it was used.

Reply:
Understand, makes sense.